src/common/components/PageLoader.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Backdrop, CircularProgress } from '@mui/material'
+
+const PageLoader = () => (
+  <Backdrop
+    sx={{ color: 'white', zIndex: theme => theme.zIndex.drawer + 1 }}
```

Comment:
Would white be the default backdrop?

Reply:
@paulschreiber Yes, for this one I just extracted all the duplciated code for the Backdrop, nothing is changing

Reply:
Would it work without specifying the color?

Reply:
@paulschreiber It works, the loader jsut displays black, I can simplify it to that since there is no erquirement for that white, let me know what do you think